### DISCUSSION TITLE
Missing device in db

### DISCUSSION THREAD

**Discussion by kjellml80** [zwave-js/zwave-js#6053] [General] [answered]
Hi

I have a zwave device which is not properly recognized in z-wave js.

Manufacturer: Namron AS
Product: 0x0004
Product code: Unknown product 0x0108
FW: FW: v10.22.3 SDK: v7.16.3

Link to product documentation: https://www.elektroimportoren.no/docs/lib/4512757-Manual-18.pdf

Any hope to get this recognized properly?

**Comment by kpine** [comment, marked as answer]
Submit an issue. https://github.com/zwave-js/node-zwave-js/issues/new/choose

![image](https://github.com/zwave-js/node-zwave-js/assets/791794/93957860-e706-4d82-94cd-4316adac194b)
